.drone.star

```diff
@@ -228,6 +229,46 @@ def buildAndPublishDocker():
     ]
   }
 
+def coverage():
+  return {
+    "kind": "pipeline",
+    "type": "docker",
+    "name": "unit-test-coverage",
+    "platform": {
+      "os": "linux",
+      "arch": "amd64",
+    },
+    "trigger":{
+      "branch":[
+        "master"
```

Comment:
@ishank011 master should now also run the coverage report

Reply:
Should we change `event` to `push`?

Reply:
I'm not sure why we need push because PR and master should be the only cases!?

Reply:
`"trigger": { "ref": [ "refs/heads/master", "refs/pull/**", ], },` is working fine in all owncloud repos